test/tests/cht/nondimensional/nek_master.i

```diff
@@ -76,31 +76,27 @@
   [temperature]
     type = MultiAppNearestNodeTransfer
     source_variable = temp
-    direction = from_multiapp
-    multi_app = nek
+    from_multi_app = nek
     variable = nek_temp
   []
   [flux]
     type = MultiAppNearestNodeTransfer
     source_variable = flux
-    direction = to_multiapp
-    multi_app = nek
+    to_multi_app = nek
     variable = avg_flux
     source_boundary = '1'
   []
   [flux_integral]
     type = MultiAppPostprocessorTransfer
     to_postprocessor = flux_integral
-    direction = to_multiapp
```

Comment:
Just curious, how does this work now? Here for example the postprocessor has the same name in parent + sub, how is the direction deduced?

Reply:
Oh I didn't look close enough, I see `to/from_multi_app` now. Like the syntax!